### PR TITLE
Db week2

### PR DESCRIPTION
**Pull request by saidcetinkaya**

### CODE REVIEW

File: Week2/homeworkWeek2.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  prompt.start();
+  try {
+    const use_database = `use world`;
+    await execQuery(use_database);
+
+    const options = await input(['number']);
+    const inputNumber = options.number;
+
+    switch (inputNumber) {
+      case '1':
```
Comment (unmeshvrije) [inline Week2/homeworkWeek2.js]: I like the use of switch case, but user has to run the program multiple times to execute different queries. After every input, the program terminates. Writing a menu driven program would be a good idea. (having a while loop around the switch case and providing an option for executing) Check this link to know more about menu driven program : https://www.geeksforgeeks.org/menu-driven-program-using-switch-case-c/

File: Week2/homeworkWeek2.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+        const inputLanguage = regionAndLanguage.language;
+        const fourthQuery = `SELECT country.name FROM country 
+                              JOIN countrylanguage ON country.code = countrylanguage.countrycode
+                              WHERE countrylanguage.isofficial = 'T' AND country.region = ? AND countrylanguage.language = ?;
+                              `;
+        const fourthResults = await execQuery(fourthQuery, [inputRegion, inputLanguage]);
+        console.log(
+          `The countries in which ${inputLanguage.toUpperCase()} is spoken in the ${inputRegion.toUpperCase()} are:`,
+        );
+        if (fourthResults.length === 0) {
```
Comment (unmeshvrije) [inline Week2/homeworkWeek2.js]: I like that you handled it in the JavaScript part. One could write a stored procedure in SQL for this. But this is more elegant.

**Comment by unmeshvrije** [comment]
reviewed.

**Comment by unmeshvrije** [comment]
reopening for changes.

**Comment by wouterkleijn** [comment]
Thanks @unmeshvrije Can you please address the changes @saidcetinkaya ?

**Comment by saidcetinkaya** [comment]
@unmeshvrije I added trigger and todo diagram.

**Comment by wouterkleijn** [comment]
Thanks @saidcetinkaya Can you have a look @unmeshvrije ?

**Pull request closed without merging**